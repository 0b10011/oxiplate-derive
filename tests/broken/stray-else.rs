use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{{ action }}{%- else -%}"]
struct Data {
    action: &'static str,
}

fn main() {
    let data = Data {
        action: "do something",
    };

    print!("{}", data);
}
