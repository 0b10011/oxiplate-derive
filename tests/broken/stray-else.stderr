error: Unexpected 'else' statement
 --> tests/broken/stray-else.rs:4:38
  |
4 | #[oxiplate_inline = "{{ action }}{%- else -%}"]
  |                                      ^^^^
